src/decision/mcda_methods.jl

```diff
@@ -98,27 +98,28 @@ Details of this aggregation method in, for example [1]
     https://doi.org/10.3390/jrfm14060271
 
 # Arguments
-- `S` : Matrix
+- `S` : Decision matrix
 - `v` : Real
 
 # Returns
 - `Q` : aggregate score for ranking.
 """
-function adria_vikor(S::Array{Float64,2}; v::Float64=0.5)::Array{Float64}
-
-    # Remove columns with constant values. Zero-valued columns will return NaNs in the Q score.
-    S = S[:, .!vec((all(S .== maximum(S; dims=1); dims=1)))]
-    F_s = maximum(S)
+function adria_vikor(S::Matrix{Float64}; v::Float64=0.5)::Array{Float64}
+    # Remove columns with constant values.
+    # Zero-valued columns will return NaNs in the Q score.
+    if size(S, 1) > 1
```

Comment:
So there are cases where an empty matrix is not caught by the outer For loop? - https://github.com/open-AIMS/ADRIA.jl/blob/0667a6b6235bc2eeb7e7cc2d85bd96296c1b2b9e/src/decision/dMCDA.jl#L695

Reply:
This catches cases where the matrix has 1 row, not when it is empty.

If it has 1 row, then this:
```julia
S = S[:, .!vec((all(S .== maximum(S; dims=1); dims=1)))]
```

will remove it (because the maximum value is the only value), which then leads to an error.

Reply:
Ah I see, thanks!